thingsboard_gateway/connectors/mqtt/mqtt_connector.py

```diff
@@ -215,16 +215,21 @@ def __init__(self, gateway, config, connector_type):
             "name",
             'Mqtt Broker ' + ''.join(random.choice(string.ascii_lowercase) for _ in range(5))))
 
+        self.__log.debug("Connector connecting to broker with security: %s", self.__broker["security"])
```

Comment:
Could you please remove this log message? Because some users during copying logs can forget to remove it from logs and will share their credentials.